src/api/routes/password.reset.js

```diff
@@ -0,0 +1,29 @@
+import { Router } from 'express';
+import resetController from '../controllers/password.reset';
+import validateInputs from '../../middlewares/validations/body.inputs';
+import verifyBody from '../../middlewares/body.verifier';
+import validateResetEmail from '../../middlewares/validations/validate.reset.email';
+import validateResetLink from '../../middlewares/validations/validate.reset.link';
+import validateUpdatePassword from '../../middlewares/validations/validate.update.password';
+
+const router = Router();
+
+router.post(
```

Comment:
@nkpremices Any reason why is this a POST instead of a GET request?

Reply:
Because I expect an email from the user